common/source_model/src/main/scala/weco/catalogue/source_model/sierra/rules/SierraItemAccess.scala

```diff
@@ -40,6 +42,41 @@ object SierraItemAccess extends SierraQueryOps with Logging {
     bibStatus: Option[AccessStatus],
     location: Option[PhysicalLocationType],
     itemData: SierraItemData
+  ): (Option[AccessCondition], Option[String], ItemStatus) =
```

Comment:
This return type is complex enough to warrant its own type (rather than being a tuple). I don't think that needs to be done in this PR, but I may pick that up when I come back and remove `ItemStatus`.